core/src/main/scala/akka/kafka/ConsumerFailed.scala

```diff
@@ -28,6 +28,11 @@ class InitialPollFailed(val timeout: Long, val bootstrapServers: String)
       s"$bootstrapServers did not succeed, correct address?"
     )
 
+class ConsumerCreationFailed(val timeout: FiniteDuration)
```

Comment:
final?

Reply:
It is not even used anymore, removed it.